File: examen/Ejercicio4.py
def ejercicio4(): 
  #Definir varibles
  print("calcule su operacion aritmetica")
  operacionJNAQ=0
  #Datos de entrada
  numeroJNAQ=int(input("Ingrese el primer numero:"))
  numero1JNAQ=int(input("Ingrese el segundo numero:"))
  signoJNAQ=input("ingrese el signo:")
  #proseso
  if signoJNAQ=="+":
    operacionJNAQ=(numeroJNAQ+numero1JNAQ)
  if signoJNAQ=="-":
    operacionJNAQ=(numeroJNAQ-numero1JNAQ)
  if signoJNAQ=="*":
    operacionJNAQ=(numeroJNAQ*numero1JNAQ)
  if signoJNAQ=="/": 
    operacionJNAQ=(numeroJNAQ/numero1JNAQ)
  if signoJNAQ=="^": 
    operacionJNAQ=(numeroJNAQ^numero1JNAQ)
    #datos de salida
  print("El resultado de la operacion es:",operacionJNAQ)
ejercicio4()

File: trabajo2/Ejercicio4.java
import java.util.Scanner;

public class Ejercicio4 {
    static Scanner teclado=new Scanner(System.in);
    public static void contadorf() {
   //definir variables y Otros
   int ncantidad,rojo=0,verde=0,blanco=0;
   String numero;
   //datos de entrada

  System.out.println("ingrese la cantidad de unidades del lote:");
   ncantidad=teclado.nextInt();
   //proseso
   int contador=1;
   while (contador<=ncantidad) {
       System.out.println("ingrese el color de la unidad "+contador+" del lote:");
       numero=teclado.next();
        if  (numero.equals("R") ) {
        rojo+=1;} 
        if (numero.equals("V")) {
        verde+=1; }
        if (numero.equals("B")) {
        blanco+=1;}
    contador++;
            
   }
   //datos de salida
   System.out.println("el resultado es:");
   System.out.println(rojo+"=rojos");
   System.out.println(verde+"=verdes");
   System.out.println(blanco+"=blancos");
    }
public static void main(String[] args) {
    contadorf();
}  
}